c-vendor/vendor.sh

```diff
@@ -8,11 +8,12 @@ if [ ! -d ./squashfuse ]; then
     git clone https://github.com/vasi/squashfuse
 fi
 
-# This is just tip/master as of Aug 30th 2021, there is no other
-# specific reason to use this. It works with both "libfuse-dev" and
-# "libfuse3-dev" which is important as 16.04 only have libfuse-dev
-# and 21.10 only has libfuse3-dev
+# This is the commit that was tagged as 0.2.0, released on June 2023:
+# https://github.com/vasi/squashfuse/releases/tag/0.2.0
+# It contains bug fixes and adds multithreading support to squashfuse_ll.
+# It still should work with both "libfuse-dev" and "libfuse3-dev" which
+# is important as 16.04 only has libfuse-dev and 21.10 only has libfuse3-dev
 if [ -d ./squashfuse/.git ]; then
-    (cd squashfuse && git checkout 74f4fe86ebd47a2fb7df5cb60d452354f977c72e)
+    (cd squashfuse && git checkout 7ce9d15f4b0a7a76ddf08e662abde4a3e340bb41)
```

Comment:
Given that this is now properly tagged, should we use `git checkout 0.2.0` here instead?

Reply:
I thought that you maybe wanted an immutable reference, but it's up to you if you want to use tags.

Reply:
Thanks, yeah, this is a good point, let's keep it like it is. There is always the risk of someone force-pushing over a tag.